pages/contact/index.tsx

```diff
@@ -0,0 +1,15 @@
+import type { NextPage } from 'next';
+import NavBar from "../../lib/components/navbar";
+import navLinks from "../../lib/components/navbar/links";
+
+const ContactUs: NextPage = () => {
+
+    return (
+      <>
+      <NavBar navLinks={navLinks} />
```

Comment:
❓ This looks suspiciously not formatted correctly, seems like we want another level of indentation on this line. Did you run prettier?